torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9210,6 +9212,9 @@ def ref_pairwise_distance(input1, input2):
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_diagonal_diag_embed),
     OpInfo('diagonal',
+           # They are not strictly aliases, but we can see them as aliases for testing purposes
```

Comment:
How are they not "strictly" aliases?

Reply:
Becuase this one  has `-2, -1` as the default dims (good) while `torch.diagonal ` has `0, 1` as default dims (ugh).

Reply:
Added a comment.